src/NServiceBus.Core/DelayedDelivery/TimeoutManager/TimeoutRecoverabilityBehavior.cs

```diff
@@ -7,6 +7,7 @@ namespace NServiceBus
     using System;
     using System.Collections.Generic;
     using System.Threading.Tasks;
+    using JetBrains.Annotations;
```

Comment:
Just thinking that as we move away from R# (like we do with the code analysers built by @distantcam and @WilliamBZA ) things like this will be more and more "noise"? do we add similar stuff for code rush, omnisharp, etc?


@Particular/nservicebus-maintainers thoughts? ( sorry for derailing this pull @MarcinHoppe )